runtime-light/stdlib/zlib/zlib-functions.cpp

```diff
@@ -36,6 +37,21 @@ voidpf zlib_static_alloc(voidpf opaque, uInt items, uInt size) noexcept {
 
 void zlib_static_free([[maybe_unused]] voidpf opaque, [[maybe_unused]] voidpf address) noexcept {}
 
+voidpf zlib_dynamic_alloc([[maybe_unused]] voidpf opaque, uInt items, uInt size) noexcept {
+  auto* mem{kphp::memory::script::calloc(items, size)};
```

Comment:
Could you please explain the reason you chose `calloc` instead of `alloc`?

Reply:
Because old KPHP runtime does